src/Components/ArticleDetailPage.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
+import axios from 'axios'
+import './ArticleDetailPage.css'
+
+function ArticleDetailPage() {
+    const { article_id} = useParams()
+    const [article, setArticle] = useState(null)
+    const [loading, setLoading] = useState(true)
```

Comment:
well done for having a loading state to provide visual feedback to the user as well as an error state!